indra/llimage/llimagedxt.cpp

```diff
@@ -207,10 +207,7 @@ bool LLImageDXT::updateData()
 		height = header->maxheight;
 	}
 
-	if (data_size < mHeaderSize)
-	{
-		LL_ERRS() << "LLImageDXT: not enough data" << LL_ENDL;
-	}
+	llassert_always_msg(data_size >= mHeaderSize, "LLImageDXT: not enough data");
```

Comment:
I thing this is a sign of memory correption, thus should error globaly, but otherwise it should set setLastError("Invalid input"); or something like that.